lib/active_interaction/inputs.rb

```diff
@@ -175,7 +176,7 @@ def normalize(inputs)
     end
 
     def convert(inputs)
-      return inputs.stringify_keys if inputs.is_a?(Hash)
+      return inputs.stringify_keys if inputs.is_a?(Hash) || inputs.is_a?(Inputs)
```

Comment:
We shouldn't need to stringify the keys.

```suggestion
      return inputs if inputs.is_a?(self.class)
      return inputs.stringify_keys if inputs.is_a?(Hash)
```

Reply:
Ah yes, good point!